Sources/TuistDependenciesTesting/DependenciesGraph/DependenciesGraph+TestData.swift

```diff
@@ -381,6 +391,7 @@ public extension TuistCore.DependenciesGraph {
                             ],
                             settings: Settings(
                                 base: [
+                                    "FRAMEWORK_SEARCH_PATHS": "$(PLATFORM_DIR)/Developer/Library/Frameworks",
```

Comment:
might make sense to add some helper method/constant to add settings that is shared across the test data, so these changes become a little bit less repetitive

Reply:
Yep, much better 🙌